config/webpack/shared.js

```diff
@@ -136,6 +136,7 @@ module.exports = {
       'bootstrap-select': '@pf3/select', // never use vanilla bootstrap-select
       '@patternfly/patternfly': resolveModule('NONEXISTENT'),
       '@patternfly/patternfly-next': resolveModule('NONEXISTENT'),
+      '@@MiQ-DDF': resolve(dirname(__filename), '../../app/javascript/forms/data-driven-form'),
```

Comment:
what about just `@@ddf `

nobody will ever type MiQ with the right capitalization

Reply:
And you may need a similar entry in `jest.config.js` - `moduleNameMapper` section

Reply:
Updated